stock_picking_report_custom_line_order/README.rst

```diff
@@ -0,0 +1,36 @@
+**This file is going to be generated by oca-gen-addon-readme.**
```

Comment:
Could you provide a minimalist readme/DESCRIPTION.rst file as OCA convention use it to properly render this file automatically while merging (and/or using pre-commit hooks)